src/brevitas/core/function_wrapper/ops_ste.py

```diff
@@ -38,6 +40,28 @@ def forward(self, x: torch.Tensor):
         return floor_ste(x)
 
 
+class AdaRoundSte(brevitas.jit.ScriptModule):
+    """
+    This Module implements AdaRound representation, where each weight has a learnable parameter
+    that decides if "ceil" or "floor" rounding type has to be used.
+    """
+
+    def __init__(self, tracked_parameter_list: List[torch.nn.Module], adaround_func: Callable[[torch.Tensor, float], torch.Tensor], temperature: float = 1.0) -> None:
+        super(AdaRoundSte, self).__init__()
+        if len(tracked_parameter_list) > 1:
+            raise RuntimeError('AdaRound does not support shared quantizers')
+        self.adaround_func = adaround_func
+        self.temperature = temperature
```

Comment:
Shall we change this to adaround_temperature because of DepInj?

Reply:
yes